content/docs/tour/programs.md

```diff
@@ -1,27 +1,3 @@
 ---
-title: Beyond the Basics
-expanded_url: /docs/tour/programs/
-menu:
-  tour:
-    identifier: programs
-    weight: 2
+redirect_to: /docs/reference/concepts/
```

Comment:
More of a question: Are the tour pages now covered under "Getting Started" and "Architecture & Concepts" pages?

Reply:
The goal of "Getting Started" is to teach simple Pulumi concepts along the way. I'm sure there's more we could do to improve this. The "Architecture & Concepts" would go into all the details.

The problem with redirecting to the Getting Started pages from the tour pages: The main tour index could link to `/docs/quickstart`, but for individual tour pages, which Get Started cloud would you link too?